src/core/common/settings.cpp

```diff
@@ -33,6 +33,8 @@
 
 #include "settings.hpp"
 
+#include "common/code_utils.hpp"
+#include "common/error.hpp"
```

Comment:
Remove these new `#include`?

Everything is covered by `instance.hpp` already.

Reply:
fixed.